src/metabase/api/public.clj

```diff
@@ -317,8 +317,14 @@
                                                         [:dashcard-id   ms/PositiveInt]
                                                         [:card-id       ms/PositiveInt]
                                                         [:export-format (into [:enum] api.dataset/export-formats)]]
+   _query-parameters
    {:keys [format_rows pivot_results parameters]} :- [:map
-                                                      [:parameters    {:optional true} [:maybe ms/JSONString]]
+                                                      [:parameters    {:optional true} [:maybe
+                                                                                        {:decode/api
+                                                                                         (fn [x]
+                                                                                           (cond-> x
+                                                                                             (string? x) json/decode+kw))}
```

Comment:
update this to handle unencoded parameters since we want to move away from JSON-encoding in the future as discussed here https://metaboat.slack.com/archives/C010L1Z4F9S/p1738005243351179?thread_ts=1738003606.875659&cid=C010L1Z4F9S